tasks/clear-cache-reference-data.js

```diff
@@ -0,0 +1,57 @@
+const redisStore = require('../config/redis-store.js')
+
+const scanAndDelete = async (cursor, pattern) => {
+  const client = await redisStore().client
+  return await client
```

Comment:
I don't think you need the `await` if you are returning a promise in this instance.